Wright_Will/Assignments/form_validation/templates/index.html

```diff
@@ -11,20 +11,20 @@
         <div id="wrapper">
             <form class="main" action="/results" method="post">
                 <h4>Your Name:</h4>
-                <input type="text" name="name" value="">
+                <input type="text" name="name" value="{{name}}">
```

Comment:
Good. This is why, btw, flask auto-encodes text -> proper html for users automatically, and forces them to use `|safe` to disable encoding.